validator/testcases/regex.py

```diff
@@ -334,6 +362,23 @@ def rewrap():
 
 
 @register_generator
+class ExtensionManagerRegexTests(RegexTestGenerator):
+    """
+    Tests for uses of the old extension manager API, which should not be
+    referenced in new extensions.
+    """
+
+    def js_tests(self):
+        yield self.get_test(
+                r"@mozilla\.org/extensions/manager;1|"
```

Comment:
Hm. This is the indentation used for every other test in this file (except for a couple of others that I added in this PR). I think they should all be consistent, so should I leave these as-is, or fix all the others?